shared/lib_irradproc.cpp

```diff
@@ -1849,6 +1865,49 @@ int irrad::check() {
     return 0;
 }
 
+void irrad::setup_solarpos_outputs_for_lifetime(size_t ts_per_year) {
+    solarpos_outputs_for_lifetime.resize(ts_per_year);
+}
+
+bool irrad::getStoredSolarposOutputs() {
+    if (!solarpos_outputs_for_lifetime.size()) return false;
```

Comment:
Is it clearer for maintenance to use if (solarpos_outputs_for_lifetime.size() == 0) return false;

Reply:
Done